### PR TITLE
JKA-20: Initial App Styling

### PR DESCRIPTION
**Pull request by thispastwinter**
## Description

Initial Styling based on Sean's InVision design.

### 📋 Acceptance Criteria

> Checked off by the PR **Assignee(s)**

- [ ] CRITERION_HERE

### 👩‍🔬 Test Instructions

1. Run 'npm start'.
2. Check to see that the design meets the standards of Sean's Prototype.

## 🔎 Reviewer Checklist

> Checked off by the PR **Reviewers**

### Required

> These always need to be checked

- [ ] Merge destination is correct
- [ ] Code is correct as understood and conforms to quality standards
- [ ] Tests have been added where appropriate (unit, visual, end-to-end)
- [ ] Acceptance Criteria have been met

### Additional

> Delete if unneeded

- [ ] Code has been tested and confirmed locally

---

>### Roles & Responsibilities
>
>#### 👨‍💻 Justin Klaas
>
>- Initiator of this PR (be sure to set in GitHub UI)
>- Addresses feedback and change requests
>- Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
>#### 👩‍💻 Blake Tarter
>
>- Invited to review PR by **Assignee** (via GitHub UI)
>- Is expected to complete a review and address followup

### CODE REVIEW

File: src/components/App/index.scss [hunk 1]
```diff
@@ -1,32 +1,166 @@
...
+  background-color: rgb(243, 243, 243);
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
+/* LIST STYLES */
+
+.tasklist-item {
+  padding: 0 24px 0 16px;
```
Comment (blaketarter) [inline src/components/App/index.scss]: @thispastwinter Not something that you should look into changing now, but have you used `rem` or `em` units before? I find using those units for spacing/sizing a lot easier to reason about.

File: src/components/App/index.scss [hunk 2]
```diff
@@ -1,32 +1,166 @@
...
+.lists-heading {
+  padding-left: 16px;
+}
+
+.tasks-heading {	
+  margin-bottom: 16px;
+  padding-left: 9px;
+  height: 19px;	
+  color: rgba(0,0,0,0.87);	
+  font-family: 'Roboto';
```
Comment (blaketarter) [inline src/components/App/index.scss]: @thispastwinter Make sure to always define fallback fonts when you use one that comes from the network in the case that the download fails.

File: src/components/App/index.scss [hunk 3]
```diff
@@ -1,32 +1,166 @@
...
+  font-size: 2rem;
+  margin: 0 24px 0 24px;
+}
+
+/* ----------- */
+
+/* CARD STYLES */
+
+.card {
+  width: 466px;
```
Comment (blaketarter) [inline src/components/App/index.scss]: @thispastwinter What happens in the case that the screen is less than `466px`? We may want to look into these hard-coded widths once we've broken then components out.

File: src/components/App/index.scss [hunk 4]
```diff
@@ -1,32 +1,166 @@
...
+}
+
+.card-calendar {
+  height: 18px;
+  display: flex;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  box-sizing: border-box;
```
Comment (blaketarter) [inline src/components/App/index.scss]: @thispastwinter Do we need to redeclare this as `border-box`? I think in the top level global styles we're setting everything to `border-box`.

Comment (thispastwinter) [reply src/components/App/index.scss]: Haha. Not at all. That was from a copy past scenario.

File: src/components/App/index.scss (hunk 3 above)
Comment (thispastwinter) [reply src/components/App/index.scss]: Yeah, I don't normally used hard-coded widths. I think I got overwhelmed by having a design template to go off of.

**Pull request merged**